libraries/lib-music-information-retrieval/CMakeLists.txt

```diff
@@ -5,11 +5,9 @@ Library for Music Information Retrieval (MIR) algorithms.
 set( SOURCES
    GetMeterUsingTatumQuantizationFit.cpp
    GetMeterUsingTatumQuantizationFit.h
-   MirAudioReader.cpp
-   MirAudioReader.h
    MirDsp.cpp
    MirDsp.h
-   MirTypes.h
```

Comment:
Do not delete this header from CMakeLists.txt.  The build works, but the file was not deleted, and so should be made known to the IDEs as part of the project.